lib/components/WheelPicker/index.js

```diff
@@ -24,7 +24,9 @@ class WheelPicker extends Component {
      */
     style: PropTypes.oneOfType([PropTypes.object, PropTypes.number]),
     /**
-     * pass custom label style: fontSize, fontFamily
+     * pass custom label style: fontSize, fontFamily, color<br>
+     * Note: label's color will override the WheelPicker's color<br>
```

Comment:
Why is the label color overrides the picker color?

Reply:
That was the requirement as I understood it